config/owasp/suppressions.xml

```diff
@@ -16,6 +16,7 @@
     <suppress>
       <notes>jackson-databind-2.13.1.jar</notes>
       <cve>CVE-2020-36518</cve>
+      <cve>CVE-2023-35116</cve>
```

Comment:
This should already be there, pull the latest and you won't need to add this

Reply:
I have pulled the latest from R2.1 but I don't see this specific supression

Reply:
Sorry, it's a 2.0. change. Resolving.